### PR TITLE
remove TestProcessProto import from test_openclpow module

### PR DESCRIPTION
**Pull request by navjotcis**
remove the use of TestProcessProto from the test_openclpow module

### CODE REVIEW

File: src/tests/test_openclpow.py [hunk 1]
```diff
@@ -4,31 +4,30 @@
 import hashlib
 import unittest
 
-import pybitmessage.openclpow as openclpow
-
 from struct import pack, unpack
 
-from test_process import TestProcessProto
+import pybitmessage.openclpow as openclpow
```
Comment (g1itch) [inline src/tests/test_openclpow.py:9]: ```from pybitmessage import openclpow```

**Review by g1itch — CHANGES_REQUESTED** [review]
Please rebase to v0.6

**Comment by navjotcis** [comment]
rebased

**Review by PeterSurda — APPROVED** [review]
Looks ok to me

**Review by g1itch — APPROVED** [review]
OK, I'll merge

**Pull request merged**